rts/at/sra/README.md

```diff
@@ -0,0 +1,68 @@
+OVERVIEW
+--------
+An example of how to use REST-AT service.
+
+USAGE
+--------------------
+TEST SRA PARTICIPANTS
+
+Start a REST-AT coordinator on port 8080
+
+    cd <narayana-repo>/jboss-as/build/target/wildfly-28.0.0.Final
+    cp docs/examples/configs/standalone-rts.xml standalone/configuration
+    ./bin/standalone.sh -c standalone-rts.xml
+
+VERIFY THAT IT IS RUNNING, eg:
+
+    curl http://localhost:8080/rest-at-coordinator/tx/transaction-manager
+
+BUILD  AND RUN THE DEMO APPLICATION on port 8081
+
+    cd <narayana-repo>/rts/sra
```

Comment:
What did you think of my previous comment `As an aside, the directory name sra should be replaced with ...`, if you disagree with any of the reviewers comments please respond using the  `Reply...` box underneath the conversation.

Reply:
> What did you think of my previous comment `As an aside, the directory name sra should be replaced with ...`, if you disagree with any of the reviewers comments please respond using the `Reply...` box underneath the conversation.

sra is renamed to annotation, so resolving the conversation.